ir/src/main/scala/org/scalajs/core/ir/Trees.scala

```diff
@@ -816,6 +819,11 @@ object Trees {
     val tpe = NoType
   }
 
+  case class TopLevelFieldExportDef(fullName: String, field: Ident)(
+      implicit val pos: Position) extends Tree {
+    val tpe = NoType
+  }
+
```

Comment:
Right. But in that case, we can rename the tree in a binary compatible fashion. The rename does not change the binary format and the `MethodDef` only restriction is already in place, it's just not statically checked. Filed as #2725.